bokeh/themes/default.yaml

```diff
@@ -0,0 +1,192 @@
+fill_defaults:
+  fill_color: 'gray'
+  fill_alpha: 1.0
+
+line_defaults:
+  line_color: 'black'
+  line_width: 1
+  line_alpha: 1.0
+  line_join: 'miter'
+  line_cap: 'butt'
+  line_dash: []
+  line_dash_offset: 0
+
+text_defaults:
+  text_font: "helvetica"
+  text_font_size: "12pt"
+  text_font_style: "normal"
+  text_color: "#444444"
+  text_alpha: 1.0
+  text_align: "left"
+  text_baseline: "bottom"
+
+attrs:
+  Axis:
+    axis_line_color: 'black'
+    axis_line_width: 1
+    axis_line_alpha: 1.0
+    axis_line_join: 'miter'
+    axis_line_cap: 'butt'
+    axis_line_dash: []
+    axis_line_dash_offset: 0
+
+    major_tick_in: 2
+    major_tick_out: 6
+    major_tick_line_color: 'black'
+    major_tick_line_width: 1
+    major_tick_line_alpha: 1.0
+    major_tick_line_join: 'miter'
+    major_tick_line_cap: 'butt'
+    major_tick_line_dash: []
+    major_tick_line_dash_offset: 0
+
+    minor_tick_in: 0
+    minor_tick_out: 4
+    minor_tick_line_color: 'black'
+    minor_tick_line_width: 1
+    minor_tick_line_alpha: 1.0
+    minor_tick_line_join: 'miter'
+    minor_tick_line_cap: 'butt'
+    minor_tick_line_dash: []
+    minor_tick_line_dash_offset: 0
+
+    major_label_standoff: 5
+    major_label_orientation: "horizontal"
+    major_label_text_font: "helvetica"
+    major_label_text_font_size: "10pt"
+    major_label_text_font_style: "normal"
+    major_label_text_color: "#444444"
+    major_label_text_alpha: 1.0
+    major_label_text_align: "center"
+    major_label_text_baseline: "alphabetic"
+
+    axis_label: ""
+    axis_label_standoff: 5
+    axis_label_text_font: "helvetica"
+    axis_label_text_font_size: "16pt"
+    axis_label_text_font_style: "normal"
+    axis_label_text_color: "#444444"
+    axis_label_text_alpha: 1.0
+    axis_label_text_align: "center"
+    axis_label_text_baseline: "alphabetic"
+
+  BoxAnnotation:
+    fill_color: '#fff9ba'
+    fill_alpha: 0.4
+    line_color: '#cccccc'
+    line_width: 1
+    line_alpha: 0.3
+    line_join: 'miter'
+    line_cap: 'butt'
+    line_dash: []
+    line_dash_offset: 0
+
+  Circle:
+    size: 4 # XXX: Circle should be a marker, then this wouldn't be necessary.
```

Comment:
`Circle` is a marker (at least in Python) and `Marker.size` has default value set to `4`. Also this doesn't seem to me a theme defining property.